USSF-ORBIT/ussf-portal-client
# feat: Add hero image support for landing page

# SC-2636



## Proposed changes
**Should be merged after** https://github.com/USSF-ORBIT/ussf-portal-cms/pull/417
- Update query to include hero image for landing pages
- Display hero image on landing page


---



## Reviewer notes
You will need the accompanying CMS branch in order for this to work.

Testing is straightforward:
- Create a new landing page and add a hero image
- Navigate to the page in the portal and you should see your image

E2E tests:
https://github.com/USSF-ORBIT/ussf-portal/pull/350


## Setup



### Start the system
```sh
yarn services:up
yarn dev
cd ../ussf-portal-cms
yarn dev
```

Login to the portal http://localhost:3000

### Start storybook

```sh
yarn storybook
```

Login to storybook http://localhost:6006, though the command above should open it for you

---

## Code review steps

### As the original developer, I have

- [ ] Created/modified automated [E2E tests](https://github.com/USSF-ORBIT/ussf-portal)
- [ ] Followed guidelines for zero-downtime deploys, if applicable
- [ ] Use [ANDI](https://www.ssa.gov/accessibility/andi/help/install.html) to check for basic a11y issues

### As a reviewer, I have

Check out our [How to review a pull request](https://github.com/USSF-ORBIT/ussf-portal/blob/main/docs/how-to/review-pull-request.md) document.

---

## Screenshots
